bom/deployment/pom.xml

```diff
@@ -400,6 +400,11 @@
                 <artifactId>quarkus-integration-test-shared-library</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jgit-deployment</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
It shouldn't be added there. These are the test dependencies section, it should be added above.